### PR TITLE
cloud: ovirt: add note about ENGINE_API_FILTER_BY_DEFAULT

### PR DESCRIPTION
**Pull request by machacekondra**
##### ISSUE TYPE

 - Docs Pull Request

##### COMPONENT NAME

ovirt_auth

##### ANSIBLE VERSION

```
2.3.0
```

##### SUMMARY

Add a note how to setup filter level API by default for oVirt 4.1.

### CODE REVIEW

**Comment by machacekondra** [comment]
@mwperina

File: lib/ansible/modules/cloud/ovirt/ovirt_auth.py [hunk 1]
```diff
@@ -93,6 +93,9 @@
   - "In order to use this module you have to install oVirt Python SDK.
      To ensure it's installed with correct version you can create the following task:
      I(pip: name=ovirt-engine-sdk-python version=4.0.0)"
+  - "Note that in oVirt 4.1 if you want to use a user which is not administrator
```
Comment (gundalow) [inline lib/ansible/modules/cloud/ovirt/ovirt_auth.py:96]: What about in oVirt 4.0?

**Comment by gundalow** [comment]
Looks good, just one question about oVirt 4.0

File: lib/ansible/modules/cloud/ovirt/ovirt_auth.py (hunk 1 above)
Comment (machacekondra) [reply lib/ansible/modules/cloud/ovirt/ovirt_auth.py:96]: In oVirt 4.0 it's not supported, unfortunatelly.

**Review by gundalow — APPROVED** [review]

**Pull request merged**